### PR TITLE
[ECF-104] Consolidate styling in webpacker + disable asset pipeline

### PR DESCRIPTION
**Pull request by callumacrae**
### Context

Styling assets were being pulled in both through the asset pipeline and webpacker (and thus, both gems and npm), which isn't ideal.

### Changes proposed in this pull request

This PR moves the styling assets from the asset pipeline into webpacker, and removes some assets that weren't used. That also involves adding the styling+images from govspeak_publishing_assets directly into the repository, as it isn't possible to import from gems in webpacker.

I've also disabled the asset pipeline as it is no longer used at all.

### Guidance to review

Anything from vendor_styles isn't my code - it's from here with a couple minor changes: https://github.com/alphagov/govuk_publishing_components/tree/master/app/assets/stylesheets/govuk_publishing_components/components/govspeak

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -47,7 +47,6 @@ gem "tzinfo-data"
 gem "activerecord-import"
 gem "govuk-components", ">= 1.0.0"
 gem "govuk_design_system_formbuilder", "~> 2.1", ">= 2.1.5"
-gem "govuk_publishing_components", ">= 23.11.0"
```
Comment (kicferk1) [inline Gemfile:50]: Sweet. Do we need `sass-rails`? I think I added it when I added govspeak styling, on https://github.com/DFE-Digital/early-careers-framework/pull/42

**Review by kicferk1 — COMMENTED** [review]
Looks pretty good, just a few small comments, and a failing build :)

File: Dockerfile.dev [hunk 2]
```diff
@@ -59,7 +59,6 @@ COPY . .
 
 RUN yarn jest --passWithNoTests && \
     bundle exec rake assets:precompile && \
-    apk del nodejs yarn && \
```
[the remark this answers is not in the document]
Comment (callumacrae) [reply Dockerfile.dev:62]: this is step one of fixing webpack in docker. i didn't figure out the other steps but hopefully it helps someone in the future 😂

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -16,7 +16,6 @@
     <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
     <%= stylesheet_pack_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application', defer: true %>
-    <%= stylesheet_link_tag 'application', media: 'all' %>
```
[the remark this answers is not in the document]
Comment (kicferk1) [reply app/views/layouts/application.html.erb:19]: I checked it and it looks alright to me :)

[the remark this answers is not in the document]
Comment (callumacrae) [reply app/views/layouts/application.html.erb:19]: 130KB->115KB transferred
550KB->300KB of assets (significant for CPU on old mobile devices mostly)

nice :)

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**